samples/Foundatio.ReceiverConsole/packages.config

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
Can be entirely replaced by `PackageReference` in csproj.
The benefit is you can reduce number of transitive packages listed as dependencies.
E.g.: `xunit` has a dependency on `xunit.assert` and `xunit.core` which do not need to be referenced if using `PackageReference`

Reply:
If we switch to the new project system I don't think we can remove most of these references. We can do this in a second pr as well.